docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+
+## Process for starting a new translation
+
+The first step for starting a new translation is to check what exists. So far, there are repositories for the following languages:
+
+- [Spanish](https://github.com/gatsbyjs/gatsby-es)
+- [Russian](https://github.com/gatsbyjs/gatsby-ru)
+
+You can find the list of all translation requests here: https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+%22New+Translation+Request%22+
+
+> Note: Once a new translation repository is created, feel free to add it here in a PR!
+
+### Finding codeowners
+
+For a new translation, open an issue with information about your intended language. If you already have co-contributors to act as fellow [code owners](https://help.github.com/en/articles/about-code-owners) and provide checks and balances for PR reviews and quality assurance, that would be very helpful! Otherwise, you can check out other [translation request issues](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+%22New+Translation+Request%22) people have made and offer to join, or [get in touch with us](/contributing/how-to-contribute/#not-sure-how-to-start-contributing) at Gatsby for help in building your translation team.
+
+## Contributing to a translation
+
+Once a language repository is created and someone on the Gatsby team has assigned codeowners, contributions can begin. It is up to the discretion of the contributor how exactly they want to work, but it's recommended to limit the scope of PRs to 1 doc at a time to aid with code reviewing.
+
+Maintainers of a translation are expected to review translations for quality, friendly and encouraging tone of voice, and technical accuracy. These qualities should be considered in every pull request regardless of who it comes from; contributions should be encouraged from anyone in the community, not just maintainers.
+
+> ⚠️ Note: All contributors are expected to follow the [Gatsby Code of Conduct](/contributing/code-of-conduct/) and work professionally with fellow contributors. For issues with conduct, if you are unable to work things out amicably amongst yourselves (perhaps after filing a public issue or having a discussion on Discord), you can contact the Gatsby team at [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) with details about the situation.
+
+### Code review process
+
+Translation contributions must be opened as pull requests to provide time for review and comments. If actionable feedback is given on a PR review, the author must acknowledge the review and make changes with their discretion. Ignoring review feedback completely is not allowed (see response templates below for help with this).
+
+### Maintainer responsibilities
```

Comment:
We created a separate page for this in the Spanish repository, you can find it here: https://github.com/gatsbyjs/gatsby-es/blob/master/MAINTAINERS.MD

The responsibilities are almost the same, but we added some more like:

-  Have a certain amount of time weekly to dedicate to the project, around 4 to 8 hours, always remembering that we all have work and life outside the project.
- Have a good grammatic and orthography for the project. (For example in Spanish we miss a lot of accents in the words sometimes).
- Ideally, they've contributed before with the Gatsby translations. (This intended to be checked before including them as maintainers).
- A great desire to contribute to Open Source a help to grow the Gatsby community.

You can find it all in the Spanish document (probably pass Google translate on it), but the point is that I think that a `How to be a maintainer` should be included in the docs to call for new contributors, also we think that a number of maintainers between 3 and 5 are ok :thinking:

Reply:
I think having the information centrally is important so contributors don't have to go find it in a translation repo in a different language, but having it in your translation repo is helpful too. So it could live in both places 😃

Reply:
I like the responsibilities listed here. @Darking360 the point about having 4-8 hours per week I think is dependent on each language. Even with React, some languages had more work, and thus needed more time from the maintainers.